docs/general/wallets.md

```diff
@@ -41,21 +41,23 @@ Browser extension wallets at bare minimum act as key storage and management solu
 
 | Wallet                                | Browsers                                    | Staking and Nomination Pools | NFTs    | Crowdloans | Ledger support | Governance | Other features                   |
 | ------------------------------------- | ------------------------------------------- | ---------------------------- | ------- | ---------- | -------------- | ---------- | -------------------------------- |
-| [Enkrypt](https://www.enkrypt.com/)   | Brave, Chrome, Edge, Firefox, Opera, Safari | No, No                       | **Yes** | No         | **Yes**        | No         | [Enkrypt Features](#enkrypt)     |
```

Comment:
The wallets are placed in alphabetical order. I request you to revert this commit and make changes to the columns, if there is any missing information.